docs: kb on transient language code identifiers on Keyman for Windows (#1671)
keymanapp/help.keyman.com: knowledge-base/kb0116.md

    TransientLangId    REG_DWORD    0x2000                  // There is one here
    2000:{FE0420F1-38D1-4B4C-96BF-E7E20A74CFB7}{B7781350-2915-4D6F-BE76-B82B25BA4520}    REG_DWORD    0x1

HKEY_CURRENT_USER\Control Panel\International\User Profile\ko
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1255
    FeaturesToInstall    REG_DWORD    0xf7
    0412:{A028AE76-01B1-46C2-99C4-ACD9858AE02F}{B5FE1F02-D5F2-4445-9C03-C568F23C99A1}    REG_DWORD    0x1

HKEY_CURRENT_USER\Control Panel\International\User Profile\yi
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1513

HKEY_CURRENT_USER\Control Panel\International\User Profile\zh-Hans-CN
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1650
    0804:{81D4E9C9-1D3B-41BC-9E6C-4B40BF79E35E}{FA550B04-5AD7-411F-A5AC-CA038EC515D7}    REG_DWORD    0x1
```

More information on LCIDs is available in the [Microsoft LCID documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.

## Workaround
You may need to install the keyboards without a language association (or associated with EN), or remove languages that you are no longer using. 

> ### For example: With armenian.kmp

Reviewer: ```suggestion
### Associated with different language example: With armenian.kmp
```

> 1. Open Command Prompt as administrator.

Reviewer: maybe link to kb0117

Author: Yes, should I link it before the kb is available?

# HOWTO: Working with transient languages on Keyman for Windows

> This article will help solve the error for the limitation of 4 custom language tags on Keyman.
> ![Add a language to Keyboard's error](./assets/kb0116/add_language_to_keyboard_error.png)

Reviewer 2: We can't really solve the error -- it's a limitation that we need to explain. 

```suggestion
This article will explain the Windows limitation of 4 custom language tags that may result in the display of a message when you attempt to install a Keyman keyboard: "Unable to install keyboard language; Windows has a limit of 4 custom 'transient' languages, and you may have reached this limit."
![Add a language to Keyboard's error](./assets/kb0116/add_language_to_keyboard_error.png)
```

    2000:{FE0420F1-38D1-4B4C-96BF-E7E20A74CFB7}{B7781350-2915-4D6F-BE76-B82B25BA4520}    REG_DWORD    0x1

HKEY_CURRENT_USER\Control Panel\International\User Profile\ko
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1255
    FeaturesToInstall    REG_DWORD    0xf7
    0412:{A028AE76-01B1-46C2-99C4-ACD9858AE02F}{B5FE1F02-D5F2-4445-9C03-C568F23C99A1}    REG_DWORD    0x1

HKEY_CURRENT_USER\Control Panel\International\User Profile\yi
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1513

HKEY_CURRENT_USER\Control Panel\International\User Profile\zh-Hans-CN
    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1650
    0804:{81D4E9C9-1D3B-41BC-9E6C-4B40BF79E35E}{FA550B04-5AD7-411F-A5AC-CA038EC515D7}    REG_DWORD    0x1
```

### Transient LCIDs

If there are four identifiers, those are likely specified as shown:

```c
0x2000: und-Latn (Undetermined, Latin script) – often used for IPA
0x2400: tk-Arab-AF (Turkmen, Arabic script, Afghanistan) – Turkmen in Latin script is available as a non-transient language, but not Arabic script
0x2800: wbl-Arab-Af (Wakhi, Arabic script, Afghanistan)
0x2C00: grc-Grek-GR (Ancient Greek, Greek script, Greece)

> ```

Reviewer 2: There are many possible transient languages -- the usage will vary dramatically; we should remove this section.

> More on transient LCIDs in the [Published Version](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.

Reviewer 2: ```suggestion
More information on LCIDs is available in the [Microsoft LCID documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.
```

## Workaround

> You may need to install the keyboards without a language association (or associated with EN) and allow the users to setup which languages they wish to use. 

Reviewer 2: ```suggestion
You may need to install the keyboards without a language association (or associated with EN), or remove languages that you are no longer using. 
```

### For example: With armenian.kmp

1. Open Command Prompt as administrator.

2. Run the command (the final = sign is important):

    `"C:\program files (x86)\keyman\keyman desktop\kmshell.exe" -i armenian.kmp=` 

    (This will install armenian.kmp for all users, but with no language association.)
    

>     *If an error occurs, please refer to this [article](/kb/117).*

Author: Can I write another kb on this?

Reviewer 3: Maybe kb117 can go on separate PR, and move this link to that new PR
